### ISSUE TITLE
Velociraptor Loader Volume Shadow Copies

### ISSUE THREAD

**Issue by Zawadidone** [fox-it/dissect.target#184]
When using the Velociraptor artefact KapeFiles.Targets with [VSSAnalysis](https://github.com/Velocidex/velociraptor/blob/80aeffb8edff94be9ad0fae6a7dc3341fecf5e20/artifacts/definitions/Windows/KapeFiles/Targets.yaml#L33), the collection contains additional volumes as shown below:

```bash
ls Collection-HOSTNAME-TIMESTAMP/uploads/mft/

%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1  
%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy2  
%5C%5C.%5CC%3A
```

Because of this the wrong folder could be used in the [Velociraptor Loader](https://github.com/fox-it/dissect.target/blob/55aa54992cf5d69a70ec6a9cdadc59b411b8dc59/dissect/target/loaders/velociraptor.py#L27).

The current fix is to only map the C volume. But it would be nice if data in the VSS folders could be parsed additionally.

## Reference

[https://github.com/Velocidex/velociraptor/blob/80aeffb8edff94be9ad0fae6a7dc3341fecf5e20/artifacts/definitions/Windows/Collectors/VSS.yaml](https://github.com/Velocidex/velociraptor/blob/80aeffb8edff94be9ad0fae6a7dc3341fecf5e20/artifacts/definitions/Windows/Collectors/VSS.yaml)

**Comment by Zawadidone** [comment]
I wil create a PR for the current fix.

Is it supported by Dissect Target to parse data from folders that don't have a [Windows folder layout](https://github.com/fox-it/dissect.target/blob/55aa54992cf5d69a70ec6a9cdadc59b411b8dc59/dissect/target/loaders/dir.py#L83)?

```bash
%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1/:
'$Extend' [...]

%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy139/:
'$Boot'  '$Extend'  '$LogFile'  '$MFT' [...]
```

**Comment by Schamper** [comment]
Can you elaborate on your example? What is your intended goal, that those directories are added as separate "filesystems"? If that is the case, it looks like `is_drive_letter_path` is the main issue here currently, and there should be some additional flexibility added to the `find_dirs` function.

If not, can you explain what it is exactly that you're trying to achieve?

**Comment by Zawadidone** [comment]
Yes that is the case, if that means that the plugins are also executed on the additional filesystems.

**Comment by Schamper** [comment]
I believe this issue can now be closed with #185 merged?

**Comment by Zawadidone** [comment]
Yes thanks!
